### PR TITLE
Build a Product Landing Page

### PR DESCRIPTION
**Pull request by Chandan-devs-tech**
## Landing page

# In index.html I have structured a website :
- With a logo
- Using ul tag to list items.
- Added an Input box and a button to it.
# In style.css I have styled a website:
-With positioning elements.
-Adding a hover effect to the button.

### CODE REVIEW

**Review by Reggeditt — COMMENTED** [review]
nice work done!

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @Chandan-devs-tech  😃 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Good use of summary template :heavy_check_mark: 

## Required Changes ♻️
- [ ]  Awesome for using summary template, but you need to look through the lines of code and review it according to the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/review_your_project.md). At least you should add two inline comments. 😄 

- [ ] [optional] It would be great to use a couple of emojis and/or gifs in the review. They make the code review more human-friendly. 🧑‍🤝‍🧑 

Kindly check the inline comments below as well.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          </ul>
+        </nav>
+      </header>
+
+      <div class="container"></div>
+
+      <section id="hero">
+        <h2>Handcrafted, home-made masterpieces</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input name="email" id="email" type="email" placeholder="Enter your email address" required="">
```
Comment (Chandan-devs-tech) [inline index.html:27]: Please add a label tag. Label tag make the site more readable and accessible for screen readers.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        padding: 0px 20px;
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+        background-color: #eee;
+      } 
+      input[type="submit"]:hover {
+        background-color: orange;
+        transition: background-color 1s;
+      }
```
Comment (Chandan-devs-tech) [inline style.css:24]: Please make an Indentation of two spaces. This will help your code to be clean.

**Review by Chandan-devs-tech — COMMENTED** [review]
>Good job so far.
> Please check the comment below and make changes to it.

# Required changes ♻️

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @Chandan-devs-tech 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ Descriptive PR title.
✔️ You used correct Github-Flow.
✔️ Inline comments are present
✔️ The comments are very polite


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**